lib/llm-events/google-genai/chat-completion-message.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const LlmEvent = require('./event')
+const { makeId } = require('../../util/hashes')
+
+module.exports = class LlmChatCompletionMessage extends LlmEvent {
+  constructor({
+    agent,
+    segment,
+    request = {},
+    response = {},
+    index = 0,
+    message,
+    completionId,
+    transaction
+  }) {
+    super({ agent, segment, request, response, transaction })
+    this.id = makeId(36)
+    this.role = message?.role ?? 'user' // Role is only defined if it is 'model'
```

Comment:
Could it also be defined as `assistant` or `system` as well? In either case, the code makes sense.